core/arch/arm/kernel/user_ta.c

```diff
@@ -824,6 +826,130 @@ static TEE_Result set_seg_prot(struct user_ta_ctx *utc,
 	return res;
 }
 
+#ifdef CFG_UNWIND
+
+/* An item in the exception index table */
+struct unwind_idx {
+	uint32_t offset;
+	uint32_t insn;
+};
+
+#define	EXIDX_CANTUNWIND 1
+
+static uint32_t offset_prel31(uint32_t addr, int32_t offset)
+{
+	return (addr + offset) & 0x7FFFFFFFUL;
+}
+
+static TEE_Result relocate_exidx(void *exidx, size_t exidx_sz, int32_t offset)
+{
+	size_t num_items = exidx_sz / sizeof(struct unwind_idx);
+	struct unwind_idx *start = (struct unwind_idx *)exidx;
+	size_t n;
+
+	start = (struct unwind_idx *)exidx;
```

Comment:
Already done. Minor: `start` could be `const`.

Reply:
But then I would need to cast `struct unwind_idx * item = &start[n]`.